client/src/Movies/SavedList.js

```diff
@@ -1,18 +1,19 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
 
 export default class SavedList extends Component {
-  constructor(props) {
-    super(props);
-  }
-
   render() {
     return (
       <div className="saved-list">
         <h3>Saved Movies:</h3>
-        {this.props.list.map(movie => (
-          <span className="saved-movie">{movie.title}</span>
+        {this.props.list.map((movie) => (
+          <Link to={`/movies/${movie.id}`} key={movie.id}>
```

Comment:
Good stuff always membering to place a key on the first element after mapping
Keep in mind that I can add the same movie twice therefore the key will no longer be unique 
```suggestion
          <Link to={`/movies/${movie.id}`} key={movie.id + Date.now()}>
```